examples/backup/variables.tf

```diff
@@ -0,0 +1,47 @@
+variable "ibmcloud_api_key" {
+  type        = string
+  description = "The IBM Cloud API Key"
+  sensitive   = true
+}
+
+variable "region" {
+  type        = string
+  description = "Region to provision all resources created by this example"
+  default     = "us-south"
+}
+
+variable "prefix" {
+  type        = string
+  description = "Prefix to append to all resources created by this example"
+  default     = "rmq-res"
+}
+
+variable "resource_group" {
+  type        = string
+  description = "An existing resource group name to use for this example, if unset a new resource group will be created"
+  default     = null
+}
+
+variable "rabbitmq_version" {
+  type        = string
+  description = "Version of rabbitmq to deploy"
+  default     = null
+}
+
+variable "resource_tags" {
+  type        = list(string)
+  description = "Optional list of tags to be added to created resources"
+  default     = []
+}
+
+variable "access_tags" {
+  type        = list(string)
+  description = "A list of access tags to apply to the rabbitmq instance created by the module, see https://cloud.ibm.com/docs/account?topic=account-access-tags-tutorial for more details"
+  default     = []
+}
+
+variable "rmq_db_backup_crn" {
+  type        = string
```

Comment:
Can we avoid the use of rqm and rqm_db and prefer the use of rabbitmq or possibly rabbitmq_database (per fscloud module name).
Follow the example of rabbitmq_version, the variable could be rabbitmq_backup_crn.

Reply:
Will add the example later and will use rabbitmq